db/migrate/20201215191958_create_rejected_contacts.rb

```diff
@@ -0,0 +1,22 @@
+class CreateRejectedContacts < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
We need only fields from the import document + id + timestamps + imported item id + reason

Reply:
@go8soft As discussed on our call there were additional fields. Not sure about one name so will check and alter if required